src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -2066,92 +2089,27 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
             }
             break;
 
-        case IF_BI_0A:
-        case IF_BI_0B:
-        case IF_BI_0C:
-        case IF_BI_1A:
-        case IF_BI_1B:
```

Comment:
This seems reasonable to me. I guess the idea here is that for INST1, there is exactly one fmt. For INST2+, there is a "pseudo-format" in the "fmt" part of the macro that then maps to the actual format.

Reply:
But after looking into your change I believe that what you have added here will also work fine.

Reply:
> The ones named here were all of the formats that only had one possible formats. This switch insures that you properly update this method when adding new instr formats. With your change the assumption is made that a newly added instrFormat is an INST1 case by default. Where as before your chanfge it would assert on the newly added format.

That would assert only if you haven't added a new format **and** set fmt to the same value in one of emitIns_X_Y_Z functions. But if the latter is false (i.e. the function returns wrong format) then it wouldn't assert. 

After this change it is not longer required to update this switch and it will automatically check that the `fmt` set in emitIns is consistent with the one declared in instrsarm64.h table.